examples/main/main.cpp

```diff
@@ -320,8 +320,14 @@ int main(int argc, char ** argv) {
     }
 
     // prefix & suffix for instruct mode
-    const auto inp_pfx = ::llama_tokenize(ctx, "\n\n### Instruction:\n\n", add_bos);
-    const auto inp_sfx = ::llama_tokenize(ctx, "\n\n### Response:\n\n",    false);
+    std::string inp_pfx_str = "\n\n### Instruction:\n\n";
+    std::string inp_sfx_str = "\n\n### Response:\n\n";
+    if (params.escape) {
+        process_escapes(inp_pfx_str);
+        process_escapes(inp_sfx_str);
+    }
```

Comment:
I'm like 99.9% sure those have to be raw string literals, otherwise it makes no sense, escape sequences in string literals are processed at compile time, so this doesn't do anything